examples/server/public/index.html

```diff
@@ -0,0 +1,459 @@
+            text-decoration: none;
+            display: flex;
+            align-items: center;
+            cursor: pointer;
+        }
+
+        .download-btn:hover {
+            text-decoration: underline;
+        }
+
+        .hint {
+            font-size: 0.75rem;
+            color: #6b7280;
+            margin-top: 4px;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h1>TTS.cpp Server API</h1>
+        </div>
+        
+        <div class="content">
+            <form id="tts-form">
+                <div class="form-group">
+                    <label for="text-input">Text to convert</label>
+                    <textarea id="text-input" placeholder="Enter text to convert to speech..."></textarea>
+                </div>
+                
+                <div class="form-group">
+                    <button type="button" id="advanced-toggle" class="toggle-button">
+                        <span>Advanced Parameters</span>
+                        <svg id="advanced-icon" class="toggle-icon" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                            <polyline points="6 9 12 15 18 9"></polyline>
+                        </svg>
+                    </button>
+                    
+                    <div id="advanced-parameters" class="advanced-params">
+                        <div class="form-group">
+                            <label for="base-text-url">Base URL</label>
+                            <input type="text" id="base-text-url" placeholder="http://127.0.0.1:8080">
+                            <p class="hint">The address to the TTS server</p>
+                        </div>
+                        
+                        <div class="form-group">
+                            <label for="temperature">
+                                Temperature: <span id="temperature-value" class="slider-value">0.75</span>
+                            </label>
+                            <div class="slider-container">
+                                <input type="range" id="temperature" min="0" max="1" step="0.01" value="0.75" class="slider">
+                            </div>
+                            <p class="hint">Controls randomness (lower = more predictable)</p>
+                        </div>
+                        
+                        <div class="form-group">
+                            <label for="top_k">
+                                Top K: <span id="top_k-value" class="slider-value">40</span>
+                            </label>
+                            <div class="slider-container">
+                                <input type="range" id="top_k" min="1" max="100" step="1" value="40" class="slider">
+                            </div>
+                            <p class="hint">Number of most likely tokens to consider</p>
```

Comment:
Maybe add something like: "By limiting the number of tokens to consider, tokens with very low probabilities are eliminated making the output more focused and coherent."